CONTRIBUTING.md

```diff
@@ -1,48 +1,11 @@
-## Who can contribute?
-OpenRW is a community driven project, we are open to contributions from anyone
-who cares to submit a pull request. Members of the community can provide their
-input on reviewing open pull requests to make sure the changes work for them.
-Once a pull request has been vetted, a member with the appropriate rights will
-accept it into the code base.
+# How to contribute
 
-GitHub moderation privileges may be granted to active contributors who display a
-good ability to moderate in their issues and pull requests.
+To find out how to report bugs or contribute code, please read the [contributing](https://github.com/rwengine/openrw/wiki/Contributing)
+page. This page covers what types of contributions we are looking to encourage.
 
-## See what's happening
-Make sure to drop into the IRC channel #openrw on chat.freenode.net for discussion
-about features in development.
-
-## How to contribute
-If you can write source code to implement missing features or fix bugs, changes
-can be submitted for review via GitHub's pull requests system.
-
-If you aren't able to contribute source code, we are also happy to see improved
-documentation of the expected behaviour on sites like the gtamodding wiki.
-Improving this documentation makes it easier to implement the correct behaviour.
-
-## Submitting Pull Requests
-
-If you are submitting a pull request for OpenRW:
-
-1. Include a reference to any issue the pull request is intended to address.
-2. Submit unrelated changes in different pull requests.
-3. If you aren't ready to merge your changes but are looking for feedback, include "[not ready]" in the title.
-4. Changes in behaviour should be shown to match the behaviour in the original game, if applicable.
-5. Ensure that the test suite still passes after your change, updating tests if required.
-
-## Guidelines
-### What will be accepted
-* Changes that increase compatibility with the game data.
-* Fixes for technical issues, such as wide-screen or UHD resolution support.
-* Feature enhancements, such as improved support for different input schemes.
-
-### What will not be accepted
-* Changes where the ownership of the code is in question.
-* Changes to support non-GTA titles.
-
-## Evaluating pull requests
-Pull requests should be reviewed by as many people as possible for merging.
-Ideally at least two active contributors should review the changes before it is
-accepted, including the person who will accept the pull request.
+# Coding Conventions
 
+The coding conventions for OpenRW are updated regulary, and are listed on the
+[coding conventions](https://github.com/rwengine/openrw/wiki/Coding-Conventions)
+page. New code should follow these conventions to improve consistency.
```

Comment:
nit: I'd just link to this from the Contributing page about code submissions to not give the impression that contributions are restricted to code-only.
= Ditch this section